### ISSUE TITLE
Server integration tests run too soon...

### ISSUE THREAD

**Issue by ThePumpingLemma** [Sanjo/meteor-jasmine#169]
So, we have a Meteor app that has lots of calls to `Meteor.startup()`. In certain cases, we use these calls to bootstrap data in a sample database (for dev/testing purposes). Said bootstrap data is also used in some of our server integration tests. This causes a problem since the server integration tests are also run via `Meteor.startup()` and are called before the bootstrapping functions. Is there any way to force the execution of the integration tests until after the other bootstrap functions have been called?

I don't think Meteor itself provides a means for post-startup callbacks, but one horrible hack would be (in `registerFrameworks.js`):
```javascript
var startServerIntegrationTests = function () {
  if (process.env.JASMINE_SERVER_INTEGRATION !== '0' &&
      process.env.FRAMEWORK === frameworks.serverIntegration.name
  ) {
    if (__meteor_bootstrap__.startupHooks.length) {
        startServerIntegrationTests()
    } else {
        frameworks.serverIntegration.start()
    }
  }
};
Meteor.startup(startServerIntegrationTests);
```
But it would be nice if there was a better way that doesn't munge with Meteor's private variables.

**Comment by Sanjo** [comment]
See https://github.com/Sanjo/meteor-jasmine/issues/171#issuecomment-77795912

**Comment by ThePumpingLemma** [comment]
I don't think this actually solves the problem completely since I cannot guarantee that the `beforeAll()` function is called after all of the `Meteor.startup` callbacks have been called. If my initialization function depends on other startup callbacks that I don't directly control or even have access to, I cannot guarantee that my initialization function will work correctly.

**Comment by Sanjo** [comment]
Should be fixed in 0.12.5. If not please reopen.
